psiturk/api/__init__.py

```diff
@@ -237,7 +237,7 @@ def post(self):
                 func=do_approve_all,
                 trigger='interval',
                 minutes=max(int(int(data['interval']) * 60), 30),
-                next_run_time=datetime.now()
+                next_run_time=datetime.datetime.now(datetime.timezone.utc)
```

Comment:
Incorrect, should not be a second datetime here

Reply:
I'm working on this. Turns out that the APScheduler needs special timezone treatment. I'll push to your PR.